tests/test_integration_color.py

```diff
@@ -0,0 +1,77 @@
+import pytest
```

Comment:
These tests run a lot of code that is unrelated to the PR.

I'd like to focus on unit tests, for instance:

- Test a color can be marked as auto
- Test Color.parse("auto") produces an auto color
- Test setting background and color on a Styles object, produces the expected `Styles.rich_style`
- Test parsing the CSS produces the expected rules

None of the above require a running app or async context. They are less likely to break from an unrelated API change.

Reply:
With only unit tests how do we check that the "auto" takes into account the colours and opacity levels of all the Node's ancestors?

Reply:
That is a good point. I think you can create a hierarchy without running the app. Create the App, call `app.mount`.

Reply:
Yeah, keep this integration test, since it's done. But let's add some unit test for what we can easily test. We may have to solve a few such issues a `active_app` to make unit tests more practical in the future. 

Integration tests are more likely to break with API changes. And the breakage is unlikely to tell me anything I didn't know already.

Integration tests will come in to their own when the APIs are more stable. They can catch unintended API changes rather than intentional API changes.